service/role.go

```diff
@@ -129,11 +145,8 @@ func NewRoleService(cfg config.Role, token ntokend.TokenProvider) (RoleService,
 	errRetryMaxCount := defaultErrRetryMaxCount
 	if cfg.ErrRetryMaxCount > 0 {
 		errRetryMaxCount = cfg.ErrRetryMaxCount
-	}
-
-	errRetryInterval, err := time.ParseDuration(cfg.ErrRetryInterval)
-	if err != nil {
-		errRetryInterval = defaultErrRetryInterval
+	} else if cfg.ErrRetryMaxCount != 0 {
+		glg.Warnf("invalid ErrRetryMaxCount, will use default errRetryMaxCount = %d", defaultErrRetryMaxCount)
```

Comment:
should we return error here?

Reply:
fixed, https://github.com/yahoojapan/athenz-client-sidecar/pull/22/commits/74979aecb60ac0d2c4d118e4ad2564eb357674a0